src/busctl/busctl.c

```diff
@@ -1509,7 +1509,10 @@ static int message_append_cmdline(sd_bus_message *m, const char *signature, char
 
                         {
                                 unsigned i;
-                                char s[k + 1];
+                                _mallocable_(char) *s = newa(char, k + 1);
```

Comment:
hmm, so this should probably be in two lines. Mixing alloca() and variable declarations is a bit weird semantically, since you allocate two things on the stack here at the same time, once the ptr and once the char array, which I think is better done in two lines.

(doesn't really matter though)